get_dependencies.py

```diff
@@ -22,32 +40,36 @@ def download(url, filename):
                 sys.stdout.write('\r[{}{}]'.format('█' * done, '.' * (50 - done)))
                 sys.stdout.flush()
     sys.stdout.write('\n')
+    return filename
 
 
 if sys.platform.startswith('linux'):
     url = 'https://www.dropbox.com/s/ziti4nkdzhgwg1n/linux.tar.xz?dl=1'
-    name = 'linux.tar.xz'
 elif sys.platform.startswith('darwin'):
     url = 'https://www.dropbox.com/s/k4yifantsypy9xv/mac.tar.xz?dl=1'
-    name = 'mac.tar.xz'
 elif sys.platform.startswith('win32'):
     url = 'https://www.dropbox.com/s/xskj9rpn2fjkra8/win32.tar.xz?dl=1'
-    name = 'win32.tar.xz'
 
 print('[*] Downloading support files...')
-download(url, name)
 
-print('[*] Extracting files...')
-with tarfile.open(name, 'r:xz') as f:
-    f.extractall('.')
-os.remove(name)
+try:
+    name = download(url)
+    print('[*] Extracting files...')
+    with tarfile.open(name, 'r:xz') as f:
+        f.extractall('.')
+    os.remove(name)
+except ValueError as e:
```

Comment:
We don't want to catch exceptions here. If there's an exception, we want the script to fail.